build-tools/packages/build-cli/README.md

```diff
@@ -46,6 +46,7 @@ USAGE
 * [`flub merge`](docs/merge.md) - Sync branches depending on the batch size passed
 * [`flub release`](docs/release.md) - Release commands are used to manage the Fluid release process.
 * [`flub run`](docs/run.md) - Generate a report from input bundle stats collected through the collect bundleStats command.
+* [`flub setReleaseTagForPublishing`](docs/setReleaseTagForPublishing.md) - Updates the types in package.json based on the release flag. This command is intended for use in publishing pipelines and may not be intended for regular developer use.
```

Comment:
OK, last rename... :) The other commands that are used exclusively in the CI pipelines are `release fromTag`, `check buildVersion`, and `generate buildVersion`. With that in mind, let's call this `release setDtsTypes`.